packages/components/src/Disclosure/Disclosure.css

```diff
@@ -0,0 +1,69 @@
+.arrowIconWrapper {
+  transform-origin: 50% 50% 0;
+  transition: transform var(--timing-base);
+  flex-basis: 0;
+}
+
+.flippedVertical {
+  transform: rotateX(180deg);
+}
+
+.summary {
+  border: none;
+  list-style: none;
+  cursor: pointer;
+  outline-color: none;
+  transition: background var(--timing-quick), box-shadow var(--timing-base);
+}
+
+.summary::-webkit-details-marker {
+  display: none;
+}
+
+.summary:hover,
+.summary:focus {
+  background: var(--color-yellow--lightest);
+}
+
+.summary:focus,
+.summary:active {
+  box-shadow: var(--shadow-focus);
+  border: none;
+  outline: none;
+}
+
+.summary * {
+  pointer-events: none;
+}
+
+.summaryWrap {
+  align-items: baseline;
+  display: flex;
+}
+
+.summaryWrap h4 {
+  flex: 1 0 0;
+}
+
+.content {
+  display: block;
+}
+
+.details[open] > .content {
+  -webkit-animation: openAnimation var(--timing-base);
+  animation: openAnimation var(--timing-base);
+}
+
+@keyframes openAnimation {
+  0% {
+    margin-top: calc(var(--space-largest) * -1);
```

Comment:
Played around with this a bit on local and I think if we use `small` here, we can actually drop the 50% keyframe and it feels a bit smoother